apps/synapse-portal-framework/src/style/_Markdown.scss

```diff
@@ -67,6 +67,10 @@
   .text-align-center {
     text-align: center;
   }
+
+  p:last-child {
+    padding-bottom: 0;
+  }
```

Comment:
In MarkdownSynapse, remove the bottom padding from the last element if it is a `p`, fixing some minor spacing issues.